### PR TITLE
Enable adding tags on bsu

### PR DESCRIPTION
**Pull request by outscale-toa**
fixes https://github.com/outscale/terraform-provider-outscale/issues/110

### CODE REVIEW

File: outscale/resource_outscale_vm_test.go [hunk 1]
```diff
@@ -642,6 +642,10 @@ func testAccCheckOutscaleOAPIVMConfigBasic(omi, vmType, region, keypair string)
 				key   = "name"
 				value = "Terraform-VM"
 			}
+			tags {
```
Comment (outscale-mgo) [inline outscale/resource_outscale_vm_test.go:645]: I think this need to be set in the BSU :).

**Review by outscale-mgo — CHANGES_REQUESTED** [review]

**Review by outscale-mgo — APPROVED** [review]

**Comment by outscale-toa** [comment]
Merge in #409

File: outscale/resource_outscale_vm.go [hunk 2]
```diff
@@ -1049,14 +1076,59 @@ out:
...
 func resourceOAPIVMDelete(d *schema.ResourceData, meta interface{}) error {
 	conn := meta.(*OutscaleClient).OSCAPI
 
 	id := d.Id()
+	var err error
+
+	err = resource.Retry(d.Timeout(schema.TimeoutDelete), func() *resource.RetryError {
+		_, httpResp, err := conn.VmApi.StopVms(context.Background()).StopVmsRequest(oscgo.StopVmsRequest{
+			VmIds:     []string{id},
+			ForceStop: oscgo.PtrBool(true),
```
Comment (outscale-mgo) [inline outscale/resource_outscale_vm.go:1120]: Maybe there should be a way to clean stop

File: outscale/resource_outscale_vm.go [hunk 3]
```diff
@@ -1049,14 +1076,59 @@ out:
...
+		}).Execute()
+		if err != nil {
+			return utils.CheckThrottling(httpResp, err)
+		}
+		return nil
+	})
 
-	log.Printf("[INFO] Terminating VM: %s", id)
+	if err != nil {
+		return fmt.Errorf("Error Froce stopping vms before destroy %s", err)
```
Comment (outscale-mgo) [inline outscale/resource_outscale_vm.go:1129]: that message is weird.
also "s/Froce/Force/"

**Pull request merged**